### PR TITLE
'current_speed' attribute addition

### PR DESCRIPTION
**Pull request by MikeFez**
Hello!

I've been using your integration to control my ceiling fans, and it works flawlessly! One issue I've encountered is being able to determine the current speed of the fan for a variety of uses, which in my case is to update an icon on my lovelace dashboard to represent the current setting. This PR integrates the current speed as an attribute, ensuring it's updated when a command is being sent, as well as captured when updating via the bond controller.

Let me know what you think, or if any changes should be made.

- Mike

### CODE REVIEW

File: custom_components/bond/fan.py
Comment (niemyjski) [inline custom_components/bond/fan.py]: I'd set this after the call to our bond api. This way if the call fails the current reported speed will be correct and we wouldn't have to wait for the next call to update for it to get corrected.

**Review by niemyjski — CHANGES_REQUESTED** [review]
Looks good except for one call. Thanks for the PR

File: custom_components/bond/fan.py
Comment (MikeFez) [reply custom_components/bond/fan.py]: Great catch - I've updated the PR

**Review by niemyjski — APPROVED** [review]

**Comment by niemyjski** [comment]
LGTM

**Comment by MikeFez** [comment]
Wonderful! I don't have write access to the repo, so I'll let you merge it in. Thanks for the quick response!

**Comment by niemyjski** [comment]
@nguyer can I get write access :D

**Comment by niemyjski** [comment]
Thanks for the PR!

**Comment by nguyer** [comment]
Thanks guys! Appreciate the contributions!

**Pull request merged**
